pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,833 @@
+---
+permalink: /documentation/migration/
+layout: styleguide
+title: Migrating to USWDS 3.0
+category: How to use USWDS
+subnav:
+- text: Why migrate to USWDS 3.0?
+  href: '#why-migrate-to-uswds-30'
+- text: How to use this guide
+  href: '#how-to-use-this-guide'
+- text: Migration overview
+  href: '#migration-overview'
+- text: 1. Check your current USWDS code and settings versions
+  href: '#1-check-your-current-uswds-code-and-settings-versions'
+- text: 2. Integrate latest USWDS 2 changes
+  href: '#2-integrate-latest-uswds-2-changes'
+- text: 3. Install the USWDS 3.0 package
+  href: '#3-install-the-uswds-30-package'
+- text: 4. Update your Sass compiler settings and recompile CSS
+  href: '#4-update-your-sass-compiler-settings-and-recompile-css'
+- text: 5. Update to Sass module syntax
+  href: '#5-update-to-sass-module-syntax'
+- text: 6. Find which settings from your theme files you’ve customized.
+  href: '#6-find-which-settings-from-your-theme-files-youve-customized'
+- text: 7. Optimize your installation with component packages
+  href: '#7-optimize-your-installation-with-component-packages'
+---
+
+## Why migrate to USWDS 3.0?
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+**Use modern Sass syntax before the old syntax loses official support.** USWDS stylesheets are written in a language called Sass. This language updated its syntax in 2020 and the old syntax is now deprecated. Sass will no longer support the old syntax starting in October 2022. We want teams to use source code with reliable support throughout the life of their project.
```

Comment:
@bonnieAcameron @thisisdano 
Should we link to the [Sass docs](https://github.com/sass/sass/blob/main/accepted/module-system.md#timeline) that explain the deprecation timeline?